src/main/app/src/app/zaken/zaak-view/zaak-view.component.html

```diff
@@ -65,11 +65,15 @@
                                     <zac-static-text [label]="'status' | translate"
                                                      [value]="zaak.status | empty:'naam'"
                                                      [icon]="zaak.status?.toelichting ? toolTipIcon : null"
-                                                     [matTooltip]="zaak.status?.toelichting ? zaak.status.toelichting : ''"></zac-static-text>
+                                                     [matTooltip]="zaak.status?.toelichting ? zaak.status.toelichting : ''"
```

Comment:
Volgens mij is "zaak.status?.toelichting" al voldoende, want de ? is de null check